<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hcmgis_prefix_suffix_form</class>
 <widget class="QDialog" name="hcmgis_prefix_suffix_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>265</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Add Prefix/ Suffix</string>
  </property>
  <widget class="QLabel" name="LblChar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>109</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="BtnOKCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>156</width>
     <height>51</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LblInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imput Layer</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="CboInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblOutput_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field</string>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="CboField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>77</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LinPrefix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblChar_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Suffix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LinSuffix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>173</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CboCharPrefix">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>125</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>,</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>_</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>|</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>\</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>`</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>!</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>#</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>$</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&amp;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>{</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>}</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>[</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>'</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&quot;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&lt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&gt;</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LblChar_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>109</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linking Characters</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblChar_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>155</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linking Characters</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CboCharSuffix">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>173</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>,</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>_</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>|</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>\</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>`</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>!</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>#</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>$</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&amp;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>{</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>}</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>[</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>'</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&quot;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&lt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&gt;</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="ChkSelectedFeaturesOnly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>207</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected features only</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>hcmgis_prefix_suffix_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>hcmgis_prefix_suffix_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
